rpm_spec/subpackages/manageiq-core

```diff
@@ -55,12 +55,38 @@ done
 
 %files core
 %defattr(-,root,root,-)
-%{app_root}
+%{app_root}/AUTHORS
```

Comment:
Do we also need to add a `%dir %{app_root}` for the directory itself?

Reply:
so `%dir` says to add a directory only and not the files underneath it.

I was pretty sure that adding the file `app_root/AUTHORS` (in addition to the next 10
lines) would add `app_root` with the default attributes.

Reply:
Based on what I'm reading we probably need %dir for app_root, since we want the rpm to own that dir.